ai-baseline/app/utils/storage.py

```
@@ -0,0 +1,148 @@
+        return "\n".join(lines)
+    
+    @staticmethod
+    def _get_quiz_folder(rubric: Rubric) -> str:
+        """Extract quiz folder name from rubric assignment_id."""
+        return rubric.assignment_id or "unknown"
+    
+    @staticmethod
+    def save_refined_rubric(
+        rubric: Rubric,
+        assignment: Assignment,
+        rubric_file_path: Optional[str] = None,
```

Ruff (0.14.8)

Unused static method argument: `rubric_file_path`

(ARG004)